docs/user/FlowVariables.md

```diff
@@ -6,6 +6,11 @@ platform, design and tool specific variables to allow finer control and
 user overrides at various flow stages. These are defined in the
 `config.mk` file located in the platform and design specific directories.
 
+### General variables for all stages
+
+| Variable         | Description                                                                        |
+|------------------|------------------------------------------------------------------------------------|
+| `SKIP_METRICS`   | If set to 1, then metrics, report_metrics does nothing. Useful to speed up builds. |
```

Comment:
Oddly worded - is 'metrics' something different from report_metrics?

Reply:
After:

```
Log                       Elapsed seconds
1_1_yosys                       3952
1_1_yosys_hier_report           3626
2_1_floorplan                    133
2_2_floorplan_io                  14
No elapsed time found in bazel-out/k8-fastbuild/bin/logs/asap7/BoomTile/base/2_3_floorplan_tdms.log
2_4_floorplan_macro              715
2_5_floorplan_tapcell            574
2_6_floorplan_pdn                339
3_1_place_gp_skip_io             769
3_2_place_iop                     26
3_3_place_gp                    7139
3_4_place_resized                603
3_5_place_dp                    1355
4_1_cts                          635
5_1_grt                         6962
```